openlibrary/plugins/openlibrary/home.py

```diff
@@ -45,28 +45,33 @@ def GET(self):
         return page
 
 CAROUSELS_PRESETS = {
-    'preset:thrillers': '(creator:"Clancy, Tom" OR creator:"King, Stephen" OR creator:"Clive Cussler" OR creator:("Cussler, Clive") OR creator:("Dean Koontz") OR creator:("Koontz, Dean") OR creator:("Higgins, Jack")) AND !publisher:"Pleasantville, N.Y. : Reader\'s Digest Association" AND languageSorter:"English"'
+    'preset:thrillers': '(creator:"Clancy, Tom" OR creator:"King, Stephen" OR creator:"Clive Cussler" OR creator:("Cussler, Clive") OR creator:("Dean Koontz") OR creator:("Koontz, Dean") OR creator:("Higgins, Jack")) AND !publisher:"Pleasantville, N.Y. : Reader\'s Digest Association" AND languageSorter:"English"',
+    'preset:children': '(creator:("parish, Peggy") OR creator:("avi") OR title:("goosebumps") OR creator:("Dahl, Roald") OR creator:("ahlberg, allan") OR creator:("Seuss, Dr") OR creator:("Carle, Eric") OR creator:("Pilkey, Dav") OR creator:("Blume, Judy"))'
```

Comment:
To make these a little more succinct, you can also do stuff like `creator:("foo smith" OR "bar smith")`

Reply:
I think; I'm not too sure if these are equivalent, actually, but this seems to suggest they are: https://lucene.apache.org/core/2_9_4/queryparsersyntax.html#Field%20Grouping